### ISSUE TITLE
Not showing no-permission message to player that indeed do not have permission to toggle staffchat

### ISSUE THREAD

**Issue by TheJoshue** [RefracDevelopment/SimpleStaffChat2#8]
For some reason, if player do not have permission to execute toggle command (this happens only on spigot side of the plugin (i am using paper just so you know), this DOES NOT happen on BungeeCord, it's working ok on BC), the player WILL NOT receive `no-permission` message. However if player do not have permission to execute bungee simplestaffchat toggle command, he will receive `no-permission` message

So, basically players with no permission to use staffchat on spigot side, they do `/sctoggle` and literally nothing shows

**Comment by Refrac** [comment]
That was due to me forgetting to add the message to the aliases they will be added next update.

**Comment by Refrac** [comment]
This has been released: https://www.spigotmc.org/resources/simplestaffchat2-1-7-1-19.91883/update?update=461703
